<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exit_menu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Клиенты</string>
    </property>
    <addaction name="client_add_menu"/>
    <addaction name="client_search_menu"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Заказы</string>
    </property>
    <addaction name="order_add_menu"/>
    <addaction name="order_search_menu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exit_menu">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="client_search_menu">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="client_add_menu">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="stuff_add_menu">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="order_search_menu">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="order_add_menu">
   <property name="text">
    <string>Новый заказ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_menu</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
